### PR TITLE
develop-events

### PR DESCRIPTION
**Pull request by diegoveloper**
Features migrated to the new API

### CODE REVIEW

File: app/src/main/java/com/belatrixsf/connect/ui/event/detail/EventDetailActivity.java [hunk 1]
```diff
@@ -45,16 +48,19 @@
 /**
  * Created by icerrate on 27/06/2016.
  */
-public class EventDetailActivity extends BelatrixConnectActivity implements EventDetailFragmentListener {
+public class EventDetailActivity extends BelatrixConnectActivity implements EventDetailFragmentListener, FragmentListener{
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/connect/ui/event/detail/EventDetailActivity.java]: I think BelatrixConnectActivity alread have the implementation for the FragmentListener

File: app/src/main/java/com/belatrixsf/connect/ui/event/detail/EventDetailActivity.java [hunk 2]
```diff
@@ -45,16 +48,19 @@
...
     @Bind(R.id.event_picture) ImageView pictureImageView;
     @Bind(R.id.bottom_navigation) AHBottomNavigation bottomNavigation;
     @BindString(R.string.bottom_navigation_color) String navigationColor;
     private int eventId;
-    public static final int TAB_NEWS = 1;
     public static final int TAB_ABOUT = 0;
+    public static final int TAB_NEWS = 1;
+    @Bind(R.id.collapsing) CollapsingToolbarLayout collapsingToolbarLayout;
+
+
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/connect/ui/event/detail/EventDetailActivity.java]: blank spaces, why?

File: app/src/main/java/com/belatrixsf/connect/ui/event/detail/EventDetailActivity.java [hunk 3]
```diff
@@ -77,6 +83,9 @@ private void initViews() {
         bottomNavigation.setOnTabSelectedListener(new AHBottomNavigation.OnTabSelectedListener() {
             @Override
             public boolean onTabSelected(int position, boolean wasSelected) {
+
+                System.out.println("position: " + position);
+
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/connect/ui/event/detail/EventDetailActivity.java]: shouldn't have any print

File: app/src/main/java/com/belatrixsf/connect/ui/event/detail/EventDetailFragmentListener.java [hunk 4]
```diff
@@ -26,5 +26,4 @@
 public interface EventDetailFragmentListener {
 
     void showPicture(String profilePicture);
-
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/connect/ui/event/detail/EventDetailFragmentListener.java]: 😢

File: app/src/main/java/com/belatrixsf/connect/ui/event/detail/EventDetailActivity.java (hunk 2 above)
Comment (diegoveloper) [reply app/src/main/java/com/belatrixsf/connect/ui/event/detail/EventDetailActivity.java]: ( > _ < )

**Comment by diegoveloper** [comment]
Pedro, acabo de subir las correcciones que mencionaste

**Pull request merged**